tests/test_meshtastic_utils_messages.py

```
@@ -1261,3 +1443,54 @@ def test_send_text_reply_function_exists_and_callable(self):
+        packet = {
+            "decoded": {"text": "test message", "portnum": TEXT_MESSAGE_APP},
+            "fromId": "!12345678",
+            "channel": 0,
+        }
+
+        mock_interface = MagicMock()
+
+        with patch(
+            "mmrelay.db_utils.get_longname", side_effect=Exception("Database error")
+        ):
+            with patch("mmrelay.meshtastic_utils.logger"):
```

Ruff (0.15.12)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)